examples/clusterscoped/escape-to-host-si-sib-mixed.yaml

```diff
@@ -0,0 +1,35 @@
+apiVersion: intent.security.nimbus.com/v1
+kind: SecurityIntent
+metadata:
+  name: escape-to-host
+spec:
+  intent:
+    id: escapeToHost
+    description: "A attacker can breach container boundaries and can gain access to the host machine"
+    action: Block
+    params:
```

Comment:
In the params, we should have another parameter which specifies the level of the PSA. We can have  either baseline or restricted. The default value should be baseline.